components/layout/LogoutButton.js

```diff
@@ -1,11 +1,44 @@
 import Link from "next/link";
+import { useState } from "react";
+import {
+  getAuth,
+  signOut,
+  onAuthStateChanged
+} from "firebase/auth"
+import { firebaseApp } from "../../config/firebaseConfig";
 
-const LogoutButton = () => (
-  <Link href='/'>
-    <a>
-      <button type='button'>Logout</button>
-    </a>
-  </Link>
-);
 
-export default LogoutButton;
+const LogoutButton = () => {
+
+  const auth = getAuth(firebaseApp);
+  const [logoutNotification, setLogoutNotification] = useState('');
```

Comment:
Same here. You're not using the logoutNotification state anywhere in the LogoutButton component, so why do you declare it here just to ahve two lines of useless code (the other is the import for useState.

Reply:
Fixed